src/common/models/ipmodels/ipblockputtag.go

```diff
@@ -0,0 +1,37 @@
+package ipmodels
+
+import (
+	ipapisdk "github.com/phoenixnap/go-sdk-bmc/ipapi"
+	files "phoenixnap.com/pnapctl/common/fileprocessor"
+)
+
+type IpBlockPutTag struct {
```

Comment:
Why not `TagAssignmentRequest`?

Reply:
Should be addressed in most recent commit